packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -72,20 +74,28 @@ type Props = {|
     placeholder?: string,
 
     /**
-     * Whether this component is required.
-     */
-    required?: boolean,
-
-    /**
-     * The error message that should render if this field is
-     * required and left blank. Note that this message will not be used
-     * if a `validate` prop is passed in. Also note that error messages
-     * only redner for the LabeledTextField component, not for TextField.
+     * Whether this field is required to to continue, or the error message to
```

Comment:
See my earlier comment on how I think we can simplify this API in the long term.